client/devdocs/async-load/index.jsx

```diff
@@ -0,0 +1,42 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import AsyncLoad from 'components/async-load';
+import Placeholder from './placeholder';
+
+export default class DevdocsAsyncLoad extends React.PureComponent {
+	static defaultProps = {
+		placeholderCount: 5,
+	};
+	static displayName = 'DevdocsAsyncLoad';
```

Comment:
Note that because of transforms we already employ this `displayName` isn't needed - it's automatically generated from having the name on the export/class.

Reply:
Did we fix the minification bug? In the past not setting this would render something like `t` instead of the displayName on wpcalypso.wordpress.com

Reply:
I don't think so; this issue is still open, at least: #11079

Reply:
@gwwar I'm not aware of that. is it only in `wpcalypso`? I rarely ever see an actual `displayName` get set

Reply:
It only needs to be set for example components, if we're still minifying the display names

Reply:
True; I did it out of convention here but it seems unneeded as this component isn't likely to be used/shown as an example.

Reply:
Ah I didn't click through to the full diff. Yes I don't think we need it here, but we should expect to see an explicit displayName for example components.